graderx/graders/manager.py

```diff
@@ -230,10 +230,14 @@ def sanitize_and_validate_lab_data(lab_data):
         lab_data['runtime_limit'] = int(lab_data['runtime_limit'])
     except:
         raise InvalidLabDataError
-    if 'disable_internet' not in lab_data or type(lab_data['disable_internet']) != bool:
+    if 'disable_internet' not in lab_data:
```

Comment:
can we refactor all the hardcoded string?

Reply:
why don't you use `json.dumps` and use that with a configuration class or something?

e.g. make a new class that contains this config -> serialize to json -> deserialize later and use object properties instead of strings

Reply:
or use pickle

Reply:
Okay I'll, thanks alot :+1:

Reply:
I refactored all the hardcoded strings, but for the configuration class/manager I'll create a separate PR for it